app/views/payments/new.html.erb

```diff
@@ -0,0 +1,6 @@
+
+ <%# <%= render partial:'/groups/shared/nav_page', locals: {title: 'Add a spending', backurl: user_group_path(author_id:@user.id,id:@group.id)}  %> 
```

Comment:
- [ ] Please kindly delete the commented line of code to make your codebase clean and professional 🙃